assets/app/vue/components/CheckoutFlow.vue

```diff
@@ -71,14 +80,23 @@ function initPaddle(items, fn) {
     })
     .then(fn)
     .catch((error) => {
+      didReceivePaddleError.value = true;
+      errorTitle.value = 'Server Error';
       console.error(error);
```

Comment:
You'll probably wait to add a .finally() => isLoading.value = false;

Reply:
Excellent suggestion - thank you!